app/models/spotlight/dor/indexer.rb

```diff
@@ -278,6 +278,18 @@ def object_level_full_text_filenames(sdb)
       end
     end
 
+    concerning :ParkerIndexing do
+      included do
+        before_index :add_text_titles
+      end
+
+      def add_text_titles(sdb, solr_doc)
+        text_titles = sdb.smods_rec.tableOfContents.try(:content)
```

Comment:
I'm not sure whether we need to further parse the tableOfContents into multiple lines (i.e., `split(/\n/)`).

Reply:
Just read the discussion in #723 and I'm not sure either. I'll ask for more examples over there.